rose/client/game.py

```diff
@@ -4,12 +4,7 @@
 import pygame
 
 from rose.common import config, message
-import track
-import car
-import finish
-import world
-import dashboard
-import component
+from . import track, car, component, dashboard, finish, world
```

Comment:
This change is not needed, and it also harder to maintain. It is much easier to work with one import per line. If the list become too big, it show that the module has too many dependencies and it should be refactored.

You can use this format:

    from . import (
        foo,
        bar, 
        ...
    )